google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_template_constants.py

```diff
@@ -10,60 +10,82 @@ class TagTemplateConstants:
     in the tag templates.
     """
 
-    def __init__(self):
-        self.field_constants_for_cluster = self.\
-            _define_constants_for_cluster_tag_template()
-        self.field_constants_for_topic = self.\
-            _define_constants_for_topic_tag_template()
+    @classmethod
+    def get_topic_constants_list(cls):
```

Comment:
hm, I wanted to explicitly separate this method from one delivering topic fields as an object (couple of lines below). Because in some places I need an object with attributes corresponding to tag template fields and in others - an iterable of those tag template fields. And to highlight the fact that it returns an iterable, I called it get_XYZ_list(). Maybe there are more pythonic ways to achieve that goal, wdyt?

Reply:
ah, so it makes sense to leave `list` here, to differentiate with the previous one. Let's leave it as it is.